src/components/SingleList.css

```diff
@@ -28,11 +28,10 @@
 	}
 }
 
-.SingleList > Link {
-	margin-left: 0.2em;
-	width: 100%;
-	display: flex;
-	justify-content: space-between;
+.list-link {
+	margin-left: 1em;
+	color: var(--color-text);
+	text-decoration: none;
 }
```

Comment:
This works great for light mode, but it isn't visible in dark mode. Are you all planning on handling dark mode in a separate issue? I don't want to ask you to change the text color, if the background color is going to change. Currently looks like:
<img width="1470" alt="Screenshot 2024-04-03 at 2 12 40 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/4ff366d9-15b8-447b-a9d5-6401379cbdd2">

Reply:
@polly89 The background color is going to change for dark mode. Right now that issue is related to what Jose has been working with in the dark-light toggle PR and we are planning to assess it once all our current PR's have been merged so we can assess what still needs to be done.

Reply:
Sounds good. I'll approve this PR.